test/List.js

```diff
@@ -67,6 +69,10 @@ List.prototype[FL.chain] = function(f) {
     Z.concat(f(this.head), Z.chain(f, this.tail));
 };
 
+List.prototype[FL.alt] = function(other) {
```

Comment:
In [haskell](https://hackage.haskell.org/package/base-4.9.1.0/docs/src/GHC.Base.html#line-756) alt for list is concat:

```hs
instance Alternative [] where
    empty = []
    (<|>) = (++)
```